src/SFA.DAS.QnA.Database/projects/roatp/sections/section6.2.json

```diff
@@ -1,590 +1,846 @@
+                                "HintText": ""
+                            }
+                        ],
+                        "Validations": [
+                            {
+                                "Name": "Required",
+                                "Value": "",
+                                "ErrorMessage": "Tell us what type of apprenticeship training your organisation will offer"
+                            }
+                        ]
+                    },
+                    "Order": null
+                }
+            ],
+            "Complete": false,
+            "AllowMultipleAnswers": false,
+            "Order": null,
+            "Active": true,
+            "Visible": true,
+            "Feedback": null,
+            "HasFeedback": false,
+            "NotRequiredConditions": [
+                {
+                    "Field": "Apply-ProviderRoute",
+                    "IsOneOf": [
+                        "1",
+                        "3"
+                    ]
+                }
+            ],
+            "Next": [
+                {
+                    "Action": "NextPage",
+                    "ReturnId": "6030",
+                    "Conditions": [
+                        {
+                            "QuestionId": "PAT-22",
+                            "QuestionTag": null,
+                            "Contains": "Apprenticeship standards"
+                        }
+                    ],
+                    "ConditionMet": false
+                },
+                {
+                    "Action": "NextPage",
+                    "ReturnId": "6050",
+                    "Conditions": [
+                        {
+                            "QuestionId": "PAT-22",
+                            "QuestionTag": null,
+                            "Contains": "Apprenticeship frameworks"
+                        }
+                    ],
+                    "ConditionMet": false
+                }
+            ],
+            "PageOfAnswers": [],
+            "ActivatedByPageId": null,
+            "BodyText": ""
+        },
+        {
+            "PageId": "6023",
+            "SequenceId": null,
+            "SectionId": null,
+            "Title": "Type of apprenticeship training",
+            "LinkTitle": "Type of apprenticeship training",
+            "InfoText": "Type of apprenticeship training",
+            "Questions": [
+                {
+                    "QuestionId": "PAT-23",
+                    "QuestionTag": "PAT-Type-Of-Training-supporting",
+                    "Label": "What type of apprenticeship training will your organisation offer?",
+                    "ShortLabel": "",
+                    "QuestionBodyText": "<p class='govuk-body'>Select all that apply.</p>",
+                    "Hint": "",
+                    "Input": {
+                        "Type": "CheckboxList",
+                        "Options": [
```

Comment:
Looks like repeated values here in the options - are these being filtered elsewhere?

Reply:
I can't see any repeated values.  The words are different...